core/dbt/contracts/graph/model_config.py

```diff
@@ -335,6 +335,39 @@ def replace(self, **kwargs):
 @dataclass
 class SourceConfig(BaseConfig):
     enabled: bool = True
+    quoting: Dict[str, Any] = field(
```

Comment:
Should we move the addition of these new configs into a separate PR? If the goal is just to support setting `enabled` as a config on the source/table levels